pype/plugins/blender/load/load_layout.py

```diff
@@ -222,6 +222,7 @@ def update(self, container: Dict, representation: Dict):
 
         for obj in objects:
             if obj.type == 'ARMATURE':
+                if obj.animation_data and obj.animation_data.action:
```

Comment:
Seems like there is missing indentation here.

Reply:
Oh, I know why. I disabled whitespaces in GitKraken and I missed that line! Sending a fix in a minute!